py/jpy-integration/src/pythonToJava/python/test/test_jpy.py

```diff
@@ -54,7 +54,7 @@ def test_has_jpy_classes(self):
 
   def test_has_gil(self):
     PyLib = jpy.get_type('org.jpy.PyLib')
-    self.assertTrue(PyLib.hasGil())
+    self.assertFalse(PyLib.hasGil())
```

Comment:
Change method name?

Reply:
Yes, we're now asserting that _in java_ when hasGil is called, java doesnt hold the GIL.

It's either that or delete the test, which seems to just test if a util method meant for java works from py (call from py, through c, into java, then turn around and go back into c, to ask the python C API something).

Reply:
My suggestion is we should change this to `test_java_doesnt_have_gil` or something like that.